internal/crypto/cert.go

```diff
@@ -1,13 +1,19 @@
 package crypto
 
 import (
+	"bytes"
 	"crypto/x509"
 	"encoding/base64"
 	"encoding/pem"
+	"errors"
+	"os"
+	"strings"
 
 	"golang.org/x/crypto/pkcs12"
 )
 
+const certBundleKey = "certbundle"
```

Comment:
Just another open discussion, users might not be able to find a cert bundle file online. In this case, they need to download the cert files in the chain one by one and form a bundle file themselves. An alternative is to pass a dir path to akv plugin where users could put their cert files in this dir. Then, akv plugin could form a cert chain for them.

Reply:
Update: we only support PEM for this PR.